plugins/wasm_bpf/func-bpf-buffer-poll.cpp

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2019-2022 Second State INC
+
+#include "func-bpf-buffer-poll.h"
+#include "wasmedge/wasmedge.h"
+#include <shared_mutex>
+
+namespace WasmEdge {
+namespace Host {
+
+// Helper functions of context conversions.
+#define CONVTO(SIMP, INST, NAME, QUANT)                                        \
+  inline QUANT auto *to##SIMP##Cxt(QUANT INST *Cxt) noexcept {                 \
+    return reinterpret_cast<QUANT WasmEdge_##NAME##Context *>(Cxt);            \
+  }
+CONVTO(CallFrame, Runtime::CallingFrame, CallingFrame, const)
```

Comment:
If the macro is only used once, just use the raw code will improve the read ability?

Reply:
In fact the macro was copied from WasmEdge's code, so I keep it in the original form. I'll expand it now

Reply:
Fixed now